Package.swift

```diff
@@ -16,7 +16,7 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/grpc/grpc-swift.git", from: "1.23.0"),
         .package(url: "https://github.com/stephencelis/SQLite.swift.git", from: "0.15.3"),
-        .package(url: "https://github.com/Electric-Coin-Company/zcash-light-client-ffi", from: "0.10.0")
+        .package(url: "https://github.com/Electric-Coin-Company/zcash-light-client-ffi", from: "0.10.1")
```

Comment:
As I've pointed out _multiple_ times, we cannot use a `from` dependency for `zcash-light-client-ffi` because the FFI needs to exactly match, and Swift's implementation of SemVer violates the SemVer spec. Using `from` here breaks downstream consumers of the SDK when we release later tags.

Revert this change from #1493 that was not caught in review there:
```suggestion
        .package(url: "https://github.com/Electric-Coin-Company/zcash-light-client-ffi", exact: "0.10.1")
```